packages/flutter/lib/src/widgets/actions.dart

```diff
@@ -943,12 +1116,22 @@ class Actions extends StatefulWidget {
       }
       return true;
     }());
-    if (actionElement == null || action == null) {
+
+    if (actionElement == null || actions.isEmpty) {
       return null;
     }
+
     // Invoke the action we found using the relevant dispatcher from the Actions
     // Element we found.
-    return _findDispatcher(actionElement!).invokeAction(action!, intent, context);
+    final List<Object?> results = <Object?>[];
+    for (final Action<T> action in actions.keys) {
+      final ActionDispatcher dispatcher = _findDispatcher(actions[action]!);
+      results.add(dispatcher.invokeAction(action, intent, context));
+    }
+    if (results.length > 1) {
+      return results;
```

Comment:
This seems odd. How do I know which result belongs to which action?

Reply:
You don't.  I think if we do this, I'll need to invent an `ActionResults` object to hold all the results and map them to the action object that produced them or something.

Otherwise you also won't know if it was your action that returns a `List` that was run, or several actions.